ios/FluentUI/Other Cells/BooleanCell.swift

```diff
@@ -89,9 +89,7 @@ open class BooleanCell: TableViewCell {
 
     open override func didMoveToWindow() {
         super.didMoveToWindow()
-        if let window = window {
-            `switch`.onTintColor = Colors.primary(for: window)
-        }
+        `switch`.onTintColor = UIColor(dynamicColor: tokens.globalTokens.brandColors[.primary])
```

Comment:
I don't think controls should usually be accessing the global tokens directly. This should probably be something more like `tokens.switchColor`. I know that isn't in the tokens yet, and none of the colors in there are branded, but we should probably either add that or make a new subclass of TableViewCellTokens.